LICENSE

```diff
@@ -175,3 +174,28 @@
       of your accepting any such warranty or additional liability.
 
    END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
```

Comment:
I noticed that GitHub doesn't automatically identify this license.
I believe this is due to the fact that it misses this bit, which makes it a custom license.

Reply:
Yeah, I wouldn't recommend changing this file in any way from how GitHub creates it when you initialise a repo and tick the "Apache license" box.

Reply:
FWIW, I'd originally removed this section and put it in to `LICENSE_HEADER` for usage with the mycila license